internal/nostr/models.go

```diff
@@ -130,13 +130,14 @@ type OnReceiveEOSFunc func(ctx context.Context, subscription *Subscription)
 type HandleEventFunc func(event *nostr.Event, subscription *Subscription)
 
 type RequestEvent struct {
-	ID             uint
-	SubscriptionId *uint
-	NostrId        string    `validate:"required"`
-	Content        string
-	State          string
-	CreatedAt      time.Time
-	UpdatedAt      time.Time
+	ID               uint
+	SubscriptionId   *uint
+	NostrId          string    `validate:"required"`
+	Content          string
+	State            string
+	ResponseReceived bool
```

Comment:
maybe if you make it a datetime field then you know that it was received if it is filled
and you know the time of when it was received and could calculate the difference between the createat time and and the receivedat time

Reply:
Ah nice! That's a good idea!